universe/Planet.cpp

```diff
@@ -871,3 +871,23 @@ void Planet::ClampMeters() {
     UniverseObject::GetMeter(METER_REBEL_TROOPS)->ClampCurrentToRange();
     UniverseObject::GetMeter(METER_DETECTION)->ClampCurrentToRange();
 }
+
+const std::string Planet::OrdinalSuffix() const {
```

Comment:
It was, since they are so similar and the reasoning was minor, will stick with Orbital.